proto/shentu/cert/v1alpha1/cert.proto

```diff
@@ -102,12 +68,55 @@ message Library {
     string publisher = 2;
 }
 
-// CertificateProto is a proto wrapper for Certificate interface.
-message CertificateProto {
+message Compilation {
+    option (cosmos_proto.implements_interface) = "Content";
+
+    string content = 1;
+}
+
+message Auditing {
+    option (cosmos_proto.implements_interface) = "Content";
+
+    string content = 1;
+}
+
+message Proof {
+    option (cosmos_proto.implements_interface) = "Content";
+
+    string content = 1;
+}
+
+message OracleOperator {
+    option (cosmos_proto.implements_interface) = "Content";
+
+    string content = 1;
+}
+
+message ShieldPoolCreator {
+    option (cosmos_proto.implements_interface) = "Content";
+
+    string content = 1;
+}
+
+message Identity {
+    option (cosmos_proto.implements_interface) = "Content";
+
+    string content = 1;
+}
+
+message General {
+    option (cosmos_proto.implements_interface) = "Content";
+
+    string content = 1;
```

Comment:
Why are these all made to "content"? AFAIK there were different fields within them before the refactor.

Reply:
Do you mean the former "request content type" field?

Reply:
Oh right. The ReqContents had different fields within them like informations (thus not really "request content" but like "content"). Can we preserve those?